lib/commands/usermgt_api.js

```diff
@@ -128,19 +128,38 @@ const usermgtTryDeleteUser = (username) => {
  */
 const usermgtGetGroup = (name) => {
     cy.request({
-        method: "POST",
+        method: "GET",
         url: Cypress.config().baseUrl + "/usermgt/api/groups/" + name,
         followRedirect: false,
         auth: {
             'user': env.GetAdminUsername(),
             'pass': env.GetAdminPassword()
         }
     }).then((response) => {
-        expect(response.status).to.eq(201)
+        expect(response.status).to.eq(200)
         return response.body
     })
 }
 
+/**
+ * Returns the HTTP status to indicate if the group by the given name exists (HTTP 200) or not (HTTP 404).
+  * @param {String} name - The name of the group that should be retrieved.
+ */
+const usermgtGetGroupWithResponse = (name)  => {
```

Comment:
I find the function-name a bit confusing, since it only returns the response-status.
Form the name alone I would not expect this.
Maybe change it to something like `checkGroupExistsWithStatus` or just `checkGroupExists` and return true if the status is 200...

Reply:
The problem is that the whole construct returns not a usablevalue but a weird object which does not contain neither. You can see this working in https://github.com/cloudogu/backup/pull/121/files#diff-cc9009a03daa393c27d649394dcae1ec567f2cc29c07b683ef76670aa268f2a9R15 where I wanted to return a boolean which ended up as complex object with the CES IP address

Reply:
I agree with the wording, `withStatus` sound okay to me

Reply:
I reproduced what the function returns (even with a boolean return)

```javascript
   var doesGroupExist = cy.usermgtGetGroupWithGet("adminRole")
    cy.log("usermgtGetGroupWithGet() returned with", doesGroupExist)
...
    const usermgtGetGroupWithGet = (name) => {
    cy.request({
        method: "GET",
        url: Cypress.config().baseUrl + "/usermgt/api/groups/" + name,
        followRedirect: false,
        failOnStatusCode: false,
        auth: {
            'user': env.GetAdminUsername(),
            'pass': env.GetAdminPassword()
        }
    }).then((response) => {
        return response.status === 200
    })
}
```

```
{specwindow: <window>, chainerid: ch-https://192.168.56.2-65}
```

I really don't know what to do with this object